### PR TITLE
agate: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/agate.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+`agate --content {{ruta/a/contenido/}} --addr {{[::]:1965}} --addr {{0.0.0.0:1965}} --hostname {{ejemplo.com}} --lang {{sp-SP}}`
+
+- Ejecuta servidor:
+
+`agate {{path/to/file}}`
+
+- Muestra ayuda:
+
+`agate -h`
+
```
Comment (kbdharun) [inline pages.es/common/agate.md:17]:
```suggestion
```

File: pages.es/common/agate.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+> Un sencillo servidor para el protocolo de red Gemini.
+> Más información: <https://github.com/mbrubeck/agate>.
+
+- Ejecuta y genera una clave privada y un certificado:
+
+`agate --content {{ruta/a/contenido/}} --addr {{[::]:1965}} --addr {{0.0.0.0:1965}} --hostname {{ejemplo.com}} --lang {{sp-SP}}`
+
+- Ejecuta servidor:
+
+`agate {{path/to/file}}`
```
Comment (kbdharun) [inline pages.es/common/agate.md:12]:
```suggestion
`agate {{ruta/al/archivo}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

File: pages.es/common/agate.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Ejecuta y genera una clave privada y un certificado:
+
+`agate --content {{ruta/a/contenido/}} --addr {{[::]:1965}} --addr {{0.0.0.0:1965}} --hostname {{ejemplo.com}} --lang {{sp-SP}}`
+
+- Ejecuta servidor:
+
+`agate {{ruta/al/archivo}}`
+
+- Muestra ayuda:
```
Comment (navarroaxel) [inline pages.es/common/agate.md:14]:
```suggestion
- Muestra la ayuda:
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**